assets/scripts/users/localization.js

```diff
@@ -31,15 +26,6 @@ export function getUnits () {
   return store.getState().persistSettings.units
 }
 
-const SEGMENT_WIDTH_RESOLUTION_IMPERIAL = 0.25
```

Comment:
I don't think we should lose the constants. Maybe they should be moved to the `constants.js` file and exported there, but it's better to have these numbers named with something that's self explanatory (and easier to search/replace for).